<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>智能停车管理系统</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#MainWindow
{
background-image: url(:/prefix1/image/parking_mainwindow.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255,90);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="rechargePageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>充值</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="inAndOutPageButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>出入场管理</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="garagePageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>查看车库</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="queryPageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>查询历史
停车信息</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="chargeRulePageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>收费规则</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="managePageButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>75</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton:!hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(255, 255, 255，70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:hover{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,70);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }
QPushButton:pressed{
	font: 75 10pt &quot;Agency FB&quot;;
	background-color: rgb(123, 117, 69,40);
	color: rgb(242, 230, 172);
        border-width:1px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 1px 1px;
    }</string>
         </property>
         <property name="text">
          <string>管理</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="inAndOutPage">
       <widget class="QPushButton" name="inButton">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>410</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:!hover{
	background-color: rgb(123, 117, 69,30);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QPushButton:hover{
	background-color: rgb(123, 117, 69,90);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
        </property>
        <property name="text">
         <string>入场</string>
        </property>
       </widget>
       <widget class="QPushButton" name="outButton">
        <property name="geometry">
         <rect>
          <x>200</x>
          <y>410</y>
          <width>101</width>
          <height>51</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton:!hover{
	background-color: rgb(123, 117, 69,30);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QPushButton:hover{
	background-color: rgb(123, 117, 69,90);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
        </property>
        <property name="text">
         <string>出场</string>
        </property>
       </widget>
       <widget class="QLabel" name="spaceLabel">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>241</width>
          <height>141</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600; color:#318eff;&quot;&gt;车位余量&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>250</y>
          <width>271</width>
          <height>131</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>车牌号:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="vehicleIDLineEdit">
             <property name="styleSheet">
              <string notr="true">QLineEdit{
	border-width:2px;
	border-style:solid;
	border-color:gray;
	border-radius:10px;
	padding: 2px 2px;
}</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>检查口:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="ParkingLotComboBox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="billLabel">
        <property name="geometry">
         <rect>
          <x>360</x>
          <y>130</y>
          <width>301</width>
          <height>291</height>
         </rect>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;出入场信息&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButtonCheck">
        <property name="geometry">
         <rect>
          <x>250</x>
          <y>20</y>
          <width>92</width>
          <height>28</height>
         </rect>
        </property>
        <property name="text">
         <string>刷新</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="informationPage">
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QPushButton" name="AllInformatioButton">
          <property name="styleSheet">
           <string notr="true">font: 12pt &quot;仿宋&quot;;</string>
          </property>
          <property name="text">
           <string>历史停车信息</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(123, 117, 69,99);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(242, 230, 172);</string>
            </property>
            <property name="text">
             <string>车牌号:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="informationVehicleIDLineEdit">
            <property name="styleSheet">
             <string notr="true">QLineEdit{
	border-width:2px;
	border-style:solid;
	border-color:gray;
	border-radius:10px;
	padding: 2px 2px;
}</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_6">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(123, 117, 69,99);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(242, 230, 172);</string>
            </property>
            <property name="text">
             <string>日期</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QDateEdit" name="dateEdit">
            <property name="dateTime">
             <datetime>
              <hour>8</hour>
              <minute>0</minute>
              <second>1</second>
              <year>2022</year>
              <month>12</month>
              <day>31</day>
             </datetime>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="informationSearchButton">
            <property name="styleSheet">
             <string notr="true">QPushButton:!hover{
	background-color: rgb(123, 117, 69,30);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QPushButton:hover{
	background-color: rgb(123, 117, 69,90);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
            </property>
            <property name="text">
             <string>查询</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="informationTableView"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="garagePage">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="styleSheet">
             <string notr="true">background-color: rgb(123, 117, 69,99);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(242, 230, 172);</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;当前车库车辆列表&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="refreshButton">
            <property name="styleSheet">
             <string notr="true">QPushButton:!hover{
	background-color: rgb(123, 117, 69,30);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }
QPushButton:hover{
	background-color: rgb(123, 117, 69,90);
	font: 75 10pt &quot;Agency FB&quot;;
	color: rgb(0, 0, 0);
        border-width:2px;
        border-style:solid;
        border-color:gray;
        border-radius:5px;
        padding: 2px 2px;
    }</string>
            </property>
            <property name="text">
             <string>刷新</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="garageTableView"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
